src/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.csproj

```diff
@@ -3,7 +3,6 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{25A02E40-D12C-4184-B599-E4F954D142DB}</ProjectGuid>
-    <!-- UAP10.1 is not yet mapped to netstandard2.0, manually duplicate this ref -->
```

Comment:
The comment is still correct as it is why we are adding the extra uap10.1 configuration.